terraform/main.tf

```diff
@@ -66,3 +66,97 @@ resource "aws_lambda_function" "c14-team-growth-lambda" {
   }
 }
 
+data "aws_iam_role" "ecs_task_execution_role" {
```

Comment:
Perhaps consider making the security group within terraform rather than on AWS alone?

Reply:
Have added a security group for when we run the tasks, will need to be attached to the scheduler.